workbench/icons/Makefile.am

```diff
@@ -0,0 +1,20 @@
+icondir = $(datadir)/icons/hicolor/16x16/apps
```

Comment:
I had a look again and changed my opinion: the directories in geany look clean and simple so I tried starting to do it that way, @codebrainz: thanks for the link.

But I can't get it to work. In ```workbench/icons/Makefile.am``` I have put:
```
SUBDIRS = 16x16 
#24x24 32x32 48x48 scalable tango

gtk_update_icon_cache = gtk-update-icon-cache -f -t

install-data-hook:
	@-if test -z "$(DESTDIR)"; then					\
		echo "Updating Gtk icon cache.";			\
		$(gtk_update_icon_cache) "$(datadir)/icons/hicolor";	\
		$(gtk_update_icon_cache) "$(datadir)/icons/Tango"; \
	else								\
		echo "*** Icon cache not updated. Remember to run:";	\
		echo "***";						\
		echo "***   $(gtk_update_icon_cache) '$(datadir)/icons/hicolor'";\
		echo "***   $(gtk_update_icon_cache) '$(datadir)/icons/Tango'";\
		echo "***";						\
	fi

uninstall-local:
	rm -f $(DESTDIR)$(datadir)/icons/hicolor/icon-theme.cache
	rm -f $(DESTDIR)$(datadir)/icons/Tango/icon-theme.cache
```
Right now I started with the original icons only, so only subdir ```16x16``` is used. In ```workbench/icons/16x16/Makefile.am``` I have put:
```
iconsdir = $(datadir)/icons/hicolor/16x16
icons_appsdir = $(iconsdir)/apps

dist_icons_apps_DATA =							\
	workbench-bookmark.png						\
	workbench-dir.png							\
	workbench-nodirs.png						\
	workbench-project.png						\
	workbench-project-error.png
```
But when I call ```make``` I get the following error:
```
Making all in src
make[1]: Entering directory '/home/lars/DokuWiki/repos/gp-branch-workbench2/geany-plugins/workbench/src'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/home/lars/DokuWiki/repos/gp-branch-workbench2/geany-plugins/workbench/src'
Making all in icons
make[1]: Entering directory '/home/lars/DokuWiki/repos/gp-branch-workbench2/geany-plugins/workbench/icons'
 cd ../.. && /bin/bash /home/lars/DokuWiki/repos/gp-branch-workbench2/geany-plugins/missing automake-1.15 --foreign workbench/icons/Makefile
 cd ../.. && /bin/bash ./config.status workbench/icons/Makefile 
config.status: creating workbench/icons/Makefile
Making all in 16x16
make[2]: Entering directory '/home/lars/DokuWiki/repos/gp-branch-workbench2/geany-plugins/workbench/icons/16x16'
make[2]: *** No rule to make target 'all'.  Stop.
make[2]: Leaving directory '/home/lars/DokuWiki/repos/gp-branch-workbench2/geany-plugins/workbench/icons/16x16'
Makefile:509: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory '/home/lars/DokuWiki/repos/gp-branch-workbench2/geany-plugins/workbench/icons'
Makefile:578: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
```
So everything looks fine until the point where directory ```16x16``` is entered. I know make but not automake and also not the details of the geany makefile. I do not see any target in the original geany 16x16/Makefile.am so I wonder why it's not working.

I need some help here please.